src/components/pages/maintenance/jobDetails/jobDetails.js

```diff
@@ -91,13 +91,19 @@ export class JobDetails extends Component {
           t={this.props.t} />
       </ContextMenu>,
       <PageContent className="maintenance-container" key="page-content">
-        <JobGrid {...jobGridProps} />
-        <JobStatusGrid {...jobStatusGridProps} />
-        <h4 className="sub-heading">{this.props.t('maintenance.devices')}</h4>
         {
-          this.state.selectedDevices
-            ? <DevicesGrid rowData={this.state.selectedDevices} t={this.props.t} />
-            : this.props.t('maintenance.noOccSelected')
+          !this.props.error
+            ? <div>
+                <JobGrid {...jobGridProps} />
+                <JobStatusGrid {...jobStatusGridProps} />
+                <h4 className="sub-heading">{this.props.t('maintenance.devices')}</h4>
+                {
+                  this.state.selectedDevices
+                    ? <DevicesGrid rowData={this.state.selectedDevices} t={this.props.t} />
+                    : this.props.t('maintenance.noOccSelected')
```

Comment:
What is `noOccSelected`? `noOccurenceSelected`?

Reply:
Can you change it `noOccurrenceSelected`? Kind of hard to guess what is it

Reply:
Can you change it noOccurrenceSelected? Kind of hard to guess what is it

Reply:
Can you change it noOccurrenceSelected? Kind of hard to guess what is it

Reply:
Changed